CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## Unreleased
+
+### New features
+* Expose `throughput_target_gbps` and `part_size` configurations of the inner S3 client.
```

Comment:
Not blocking: Perhaps mention this is exposed through a new `S3ClientConfig` class/argument?